01-Login/README.md

```diff
@@ -6,13 +6,20 @@
 This sample demonstrates how to use Auth0 to perform authentication using the `mvc-auth-commons` library. Download or clone this repository and follow the instructions below to setup the sample.
 
 ### Auth0 Dashboard
-1. On the [Auth0 Dashboard](https://manage.auth0.com/#/clients) create a new Client of type `Regular Web Application`. 
-1. Add the URL that will be called on an OAuth successful login to the Allowed Callback URLs. i.e.: `https://mysite.com/callback`.
-1. Add the URL that will be called on logout to the Allowed Logout URLs. i.e.: `https://mysite.com`.
+1. On the [Auth0 Dashboard](https://manage.auth0.com/#/clients) create a new Application of type `Regular Web Application`. 
+1. Add the URL that will be called on an OAuth successful login to the Allowed Callback URLs. i.e.: `http://localhost:3000/callback`.
+1. Add the URL that will be called on logout to the Allowed Logout URLs. i.e.: `http://localhost:3000`.
 1. Copy the `Domain`, `Client ID` and `Client Secret` values at the top of the page and use them to configure the Java Application.
 
 
 ### Java Application
+
+Copy `src/main/resources/auth0.properties.example` to `src/main/resources/auth0.properties`:
+
+```bash
+cp src/main/resources/auth0.properties.example src/main/resources/auth0.properties
```

Comment:
what if we rename instead?
```suggestion
mv src/main/resources/auth0.properties.example src/main/resources/auth0.properties
```

Reply:
If we rename, that would show as a modified file in the git working tree, which seems like something I wouldn't want as a developer.